data_safe_haven/external/api/azure_sdk.py

```diff
@@ -581,6 +581,7 @@ def ensure_storage_account(
                     kind=StorageAccountKind.STORAGE_V2,
                     location=location,
                     tags=tags,
+                    minimum_tls_version="TLS1_2",
```

Comment:
Is there an enum from the Azure SDK we can use here instead of a raw string?

Reply:
`storage.MinimumTlsVersion`

Reply:
Is that `pulumi.azure_native`? I think this is the Python Azure SDK.

Reply:
Sorry - you're right!

Reply:
It's from `azure.mgmt.storage.v2021_08_01.models import MinimumTlsVersion`

Reply:
done